app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -188,7 +188,7 @@ def check_compliance(model)
 
   def find_current_item(model)
     if params[:id].nil? || model.find_by(:id => params[:id].to_i).nil?
-      add_flash(_("%{model} no longer exists") % {:table => ui_lookup(:model => model)}, :error)
+      add_flash(_("%{model} no longer exists") % {:table => ui_lookup(:model => model.to_s)}, :error)
```

Comment:
This seems strange. The second argument is a hash that should contain variables that appear in the first string.